xrmocap/core/estimation/mview_sperson_smpl_estimator.py

```diff
@@ -366,21 +374,33 @@ def estimate_keypoints3d(self, cam_param: List[FisheyeCameraParameter],
                     keypoints3d, **optim_kwargs)
         return keypoints3d
 
-    def estimate_smpl(self, keypoints3d: Keypoints) -> SMPLData:
+    def estimate_smpl(self,
+                      keypoints3d: Keypoints,
+                      init_smpl_dict: dict = {},
+                      return_joints: bool = False,
+                      return_verts=False) -> SMPLData:
         """Estimate smpl parameters according to keypoints3d.
 
         Args:
             keypoints3d (Keypoints):
                 A keypoints3d Keypoints instance, with only one person
                 inside. This method will take the person at
                 keypoints3d.get_keypoints()[:, 0, ...] to run smplify.
+            init_smpl_dict (dict, optional):
```

Comment:
`init_smpl_dict` is a Smplify interior data definition. As an interface, I suggest that we pass `smpl(x)_data` here, and call `init_smpl_dict = smpl_data.to_param_dict()` or `init_smpl_dict = smpl_data.to_tensor_dict()` inside `estimate_smpl()`.

Reply:
Fixed in [21a9b15](https://github.com/openxrlab/xrmocap/pull/52/commits/21a9b1598c19c0618ff463203501f7afeea2e43d).